.vscode/launch.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
#### Critical
- Explain why this file exists?

Reply:
It is coming from the debug tool. I will remove it. I thought it was a local file